storybook/.storybook/main.js

```diff
@@ -2,6 +2,7 @@ const path = require('path');
 const fs = require('fs');
 const webpack = require('webpack');
 const NM_REGEX  = /node_modules\/(.*)/
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
```

Comment:
This doesn't look like it should be part of the PR?

Reply:
@richard-cox Fixed